_datascience/data_viz.md

```diff
@@ -92,6 +94,7 @@ Data visualization focused blogs and screencasts can be a great way to find insp
 
 - [The TidyX Screencast](https://www.youtube.com/c/TidyX_screencast/featured)
 - [Flowing Data](https://flowingdata.com/)
-- [Nightingale: The journal of the Data Visualization Society](https://medium.com/nightingale)
+- [Nightingale: The journal of the Data Visualization Society](https://nightingaledvs.com)
 - [The Pudding](https://pudding.cool/)
 - [Information is Beautiful](https://informationisbeautiful.net/)
+
```

Comment:
Some other resources you can consider including:
- https://rkabacoff.github.io/datavis/
- https://www.storytellingwithdata.com/blog
- https://junkcharts.typepad.com
- https://www.economist.com/graphic-detail